.github/workflows/claim-crate-names.yml

```diff
@@ -56,6 +56,7 @@ jobs:
   claim:
     name: Claim crate names
     needs:
+    - main-branch-check
```

Comment:
Without depending on this, the two jobs race each other. `claim` will _probably_ always lose the race, but better not to chance it.